nestkernel/event_delivery_manager.cpp

```diff
@@ -69,6 +69,8 @@ EventDeliveryManager::~EventDeliveryManager()
 void
 EventDeliveryManager::initialize()
 {
+  off_grid_spiking_ =
+    false; // ensures that ResetKernel resets off_grid_spiking_
```

Comment:
I think it would be nicer to place the comment before the statement, so that the statement itself fits on a single line.